### PR TITLE
Feature/osw fixes

### PR DESCRIPTION
**Pull request by hroest**
A series of fixes and features for OpenSWATH using ion mobility:

- [FEATURE] add isolation window to Precursor
- [FEATURE] store isolation window for chromatograms 
- [FEATURE] simplify chromatogram extraction and scoring in OpenSwathWorkflow
- [FEATURE] filter SWATH Spectrum by ion mobility

### CODE REVIEW

File: src/openms/include/OpenMS/ANALYSIS/OPENSWATH/MRMTransitionGroupPicker.h [hunk 1]
```diff
@@ -473,6 +478,7 @@ namespace OpenMS
           f.setMetaValue("total_mi", transition_total_mi);
         }
 
+        // TODO shouldn't this be quantifying transition?
         if (transition_group.getTransitions()[k].isDetectingTransition())
         {
           total_intensity += peak_integral;
```
Comment (hroest) [inline src/openms/include/OpenMS/ANALYSIS/OPENSWATH/MRMTransitionGroupPicker.h]: @grosenberger can you comment on this?

Comment (grosenberger) [reply src/openms/include/OpenMS/ANALYSIS/OPENSWATH/MRMTransitionGroupPicker.h]: Yes, that seems to be an artifact. So far all detecting transitions are also quantifying transitions in our workflows, but we should change this for potential future extensions. Thanks for spotting this!

**Comment by timosachsenberg** [comment]
rebuild jenkins

**Comment by hroest** [comment]
looks like all tests pass, can we merge?

File: src/openms/include/OpenMS/ANALYSIS/OPENSWATH/MRMTransitionGroupPicker.h [hunk 2]
```diff
@@ -473,7 +484,7 @@ namespace OpenMS
           f.setMetaValue("total_mi", transition_total_mi);
         }
 
-        if (transition_group.getTransitions()[k].isDetectingTransition())
+        if (transition_group.getTransitions()[k].isQuantifyingTransition())
```
Comment (hroest) [inline src/openms/include/OpenMS/ANALYSIS/OPENSWATH/MRMTransitionGroupPicker.h]: @grosenberger I changed it now here

**Pull request merged**